packages/grid/_modules_/grid/components/panel/filterPanel/GridFilterInputValue.tsx

```diff
@@ -33,6 +33,17 @@ export interface GridTypeFilterInputValueProps extends GridFilterInputValueProps
 
 function GridFilterInputValue(props: GridTypeFilterInputValueProps & TextFieldProps) {
   const { item, applyValue, type, apiRef, focusElementRef, ...others } = props;
+  if (
```

Comment:
We only show the warning once.

https://github.com/mui-org/material-ui-x/blob/8ffe04e6efe09b17daece526d116df5eff5aa0bd/packages/grid/_modules_/grid/hooks/features/rows/useGridParamsApi.ts#L141-L142